core/src/main/java/com/dtolabs/rundeck/core/plugins/configuration/PluginAdapterUtility.java

```diff
@@ -253,6 +253,11 @@ private static Property propertyFromField(final Field field, final PluginPropert
             for (RenderingOption renderingOption : field.getAnnotationsByType(RenderingOption.class)) {
                 pbuild.renderingOption(renderingOption.key(), renderingOption.value());
             }
+        } else {
+            for (RenderingOption renderingOption : field.getAnnotationsByType(RenderingOption.class)) {
+                if(renderingOption.key().equals(StringRenderingConstants.GROUP_NAME) || renderingOption.key().equals(StringRenderingConstants.GROUPING))
```

Comment:
any reason to restrict this to only "grouping/group_name" keys? i didn't realize non-string properties were skipping this annotation, probably we should just apply it to all properties

Reply:
originally, yes, however i think we should just add them all to the properties.  if they don't make sense for that property type they will be ignored, but better to allow it to be open ended for future options